client/landing/stepper/declarative-flow/internals/steps-repository/site-creation-step/index.tsx

```diff
@@ -150,7 +150,9 @@ const SiteCreationStep: Step = function SiteCreationStep( { navigation, flow, da
 	}
 
 	useEffect( () => {
-		setProgress( 0.1 );
+		if ( ! isFreeFlow( flow ) ) {
+			setProgress( 0.1 );
+		}
```

Comment:
The way the processing screen works, the addition of `setProgress( 0.1 );` will tell the processing screen to use the loading bar. We don't want that for free screens, so exclude that here. Note that we don't need to exclude write and build flows because their site creation happens before the arrive on stepper screens and is already set up correctly.